src/Application/Migrations/23_function_encoding_profile_all_dependee.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION encoding_profile_all_dependees(param_profile_id bigint, param_known_ids bigint[] DEFAULT array[]::bigint[])
```

Comment:
Needs a description what it does.
Also it should be noted, that the second parameter is optional (and will be left empty in most use cases).